src/main/webapp/i18n/en/textExercise.json

```diff
@@ -6,6 +6,8 @@
                 "createLabel": "Create new Text Exercise",
                 "editLabel": "Edit Text Exercise"
             },
+            "wordCount": "Number of words: {{count}}",
+            "characterCount": "Number of characters: {{count}}",
```

Comment:
```suggestion
            "characterCount": "Number of characters: {{ count }}",
```

Reply:
Done in 1f009e2b14d30346d16cfb3ed30d68418a4ee8b1